lib/active_model/serializer.rb

```diff
@@ -199,16 +199,22 @@ def serializer_from_options(options)
       opts
     end
 
+    def self.serializers_cache
+      @serializers_cache ||= ThreadSafe::Cache.new
```

Comment:
`Threadsafe` is a dependancy of `ActiveSupport`, so it is already indirectly as dependancy. I can add it is explicitly as a dependancy.

The reason that to use it instead of `ActionController::Base.cache_store` is that is in purely in memory so is faster than going to an external cache like memcached which `ActionController::Base.cache_store` would do.